with_params: true
filter_by_color: true
filter_by_convexity: true
filter_by_inertia: true
filter_by_area: true
filter_by_circularity: false

color: "0"
min_area: "100"
max_area: "100000"
min_convexity: "0.8"
min_inertia: "3.40282e+38"
min_circularity: "0.8"
min_threshold: "50"
max_threshold: "220"
repeatability: "3"
#lower_1: "170"
#lower_2: "100"
#lower_3: "140"
#upper_1: "190"
#upper_2: "120"
#upper_3: "190"
lower_1: "28"
lower_2: "153"
lower_3: "51"
upper_1: "36"
upper_2: "256"
upper_3: "256"
gaussian_size_1: "11"
gaussian_size_2: "11"
gaussian_sigmax: "2"
gaussian_sigmay: "2"
houghcircles_param_1: "100"
houghcircles_param_2: "20"
